lib/Service/RecordingService.php

```diff
@@ -147,7 +147,7 @@ public function store(Room $room, string $owner, array $file): void {
 			throw new InvalidArgumentException('owner_permission');
 		}
 
-		if (!$this->serverConfig->getAppValue('spreed', 'call_recording_transcription', 'no') === 'yes') {
+		if ($this->serverConfig->getAppValue('spreed', 'call_recording_transcription', 'no') === 'yes') {
```

Comment:
```suggestion
		if ($this->serverConfig->getAppValue('spreed', 'call_recording_transcription', 'no') !== 'yes') {
```
It fixs the bug of priority, but changes the meaning.
We want to return only if config is different from `yes`, no?

Reply:
thanks, fixed by comparing to `no`